src/ray/protobuf/common.proto

```diff
@@ -170,6 +171,57 @@ message RayException {
   string formatted_exception_string = 3;
 }
 
+message ActorDeathCause {
```

Comment:
nit: enum is a bit overkill here?

Reply:
As mentioned above, this is to add additional metadata per error type. But alternatively, I can just construct the full error message from GCS and make it only have one field (error_message). Which approach do you prefer?